src/components/Personal-Zone/Searcher/AddResumeDialog/ProgressBar/Steps/PersonalDetails.tsx

```diff
@@ -36,20 +36,14 @@ const PersonalDetailsStep: React.FC<PersonalDetailsStepProps> = (props): JSX.Ele
         </MenuItem>
     )
     const rolesSelect = <Select
-        classes={{ root: classes.inputfield }}
+        className={classes.inputfield}
+        classes={{
+            root: classes.selectRoot,
+            iconOutlined: classes.selectIconOutlined,
+        }}
         disableUnderline
         value={selectedRoleId}
-        MenuProps={{
```

Comment:
why you removed that? , we dont want select go down ?

Reply:
sorry my mistake